test/unit/math/mix/mat/util/autodiff_tester.hpp

```diff
@@ -230,10 +230,10 @@ void expect_throw(const F& f, const Eigen::VectorXd& x) {
     x_t(i) = x(i);
   try {
     f(x_t);
+    FAIL() << "double throws, expect autodiff version to throw";
```

Comment:
this change looks wrong.

Reply:
Maybe I got this wrong but if it throws we expect success.  If it throws in the original code it gets caught and marked as a success, then it goes on to line 236 with regular execution and fails there (even though it met the condition for success).  With the change is succeeds when it throws and fails if it doesn't throw (after control returns from the call to `f(x_t)` on line 232.

Reply:
You're absolutely right.... which means that this function has never been used?

Reply:
Yeah, I'm not clear on why nothing was failing before... :/ maybe not used... does our testing code need unit tests? Is it turtles all the way down?

Reply:
That wasn't a serious question, I'm sure using our test code can be a fine unit test, we just can't be brain dead about writing to tests.